### PR TITLE
updating caption

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @cody-carlson
- [ ] @cody-carlson-learning
- [ ] @BRGravley
- [ ] @BrightTigger
- [ ] @CIMSims
- [ ] @DEPearson
- [ ] @DavidH102
- [ ] @GaerynDaenyr
- [ ] @bhaveshshah21-home
- [ ] @bnshawtest
- [ ] @brangull51
- [ ] @cfranchi
- [ ] @dually8
- [ ] @nunezros
- [ ] @pmconnolly80
- [ ] @rbmiller12
- [ ] @thetechlink
- [ ] @vanand11

### PR DESCRIPTION
**Pull request by BRGravley**
### Changed the header

This PR resolves / closes / fixes #2 

ToDo
- [x] Merge the branch
- [x] delete the branch afterwards

### CODE REVIEW

File: _slides/04-BRGravley.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Title for my slide
```
Comment (GaerynDaenyr) [inline _slides/04-BRGravley.md:6]: Nice!

**Review by GaerynDaenyr — APPROVED** [review]
Approved!

File: _slides/04-BRGravley.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
```
Comment (cody-carlson-learning) [inline _slides/04-BRGravley.md:6]: hey we should delete this line, good work!

**Review by cody-carlson-learning — APPROVED** [review]
Looks great! 🏅

**Pull request merged**